nx-X11/lib/X11/SetLocale.c

```diff
@@ -118,7 +118,7 @@ _Xsetlocale(
 
 #else /* X_LOCALE */
 
-#ifdef __DARWIN__
+#if defined(__APPLE__) || defined(__CYGWIN__)
```

Comment:
Hmm, upstream switched from DARWIN to APPLE in 2007 (https://cgit.freedesktop.org/xorg/lib/libX11/patch/src/SetLocale.c?id=b57129ef324c73ee91c2a796b800c4b45f4d4855), so maybe we should switch everywhere, too